tests/files.spec.ts

```diff
@@ -148,4 +149,17 @@ describe('files', () => {
     const windowsPath = 'C:\\Users\\user\\Git\\goof%20test\\index.js';
     expect(resolveBundleFilePath(baseDir, windowsPath)).toEqual(decodeURI(windowsPath));
   });
+
+  it('generates correct glob patterns for supported files', () => {
```

Comment:
enh: It would be useful to have tests for some of the weirder edge-cases, for example:
- Empty extensions
- Extensions not starting with a `.`
- Extensions with characters that don't have an uppercase variant

Reply:
Good point, although we shouldn't be returning these cases :)